databricks/koalas/base.py

```diff
@@ -57,6 +57,21 @@ def wrapper(self, *args):
             args = [arg._scol if isinstance(arg, IndexOpsMixin) else arg for arg in args]
             scol = f(self._scol, *args)
 
+            # check if f is a logistic operator
```

Comment:
I would change the comment and variable names too :-).

Reply:
Thanks! will fix them.